packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -526,8 +526,9 @@ export default class Meetings extends WebexPlugin {
    * @returns {Promise}
    * @public
    * @memberof Meetings
+   * @param {boolean} isGuest
    */
-  public register() {
+  public register(isGuest) {
```

Comment:
instead of using sGuest param use something similar to below to check guest or not
this.webex.credentials.isUnverifiedGuest

Reply:
> instead of using sGuest param use something similar to below to check guest or not this.webex.credentials.isUnverifiedGuest

By using this this.webex.credentials.isUnverifiedGuest in the function call it says that isUnverfiedGuest is undefined there is an code difference between the master and next branch @rsarika